components/Posts.php

```diff
@@ -1,56 +1,74 @@
-<?php namespace RainLab\Blog\Components;
+<?php
+
+namespace RainLab\Blog\Components;
 
 use Redirect;
 use BackendAuth;
 use Cms\Classes\Page;
 use Cms\Classes\ComponentBase;
+use October\Rain\Database\Model;
+use October\Rain\Database\Collection;
```

Comment:
Why have these been added?

Reply:
These were referenced in docblocks without proper imports and IDE complained about it.